### PR TITLE
Making the customization option docs clearer about variable overrides

### PR DESCRIPTION
**Pull request by Bobeta**
Current docs on the subject can easily be interpreted as the user should import their modified variables **after** importing bootstrap sass files, this PR aims to make it clearer #23963

### CODE REVIEW

**Comment by andresgalante** [comment]
@Bobeta Thanks for sending this PR. It' be awesome if you can add a code snippet to show exactly how to load your variables before bootstrap's. It can be something silly like:

```scss
// Variables overwrite
$body-bg: #f00;

// Bootstrap import
@import "node_modules/bootstrap/scss/bootstrap";
```
or whatever you think it's best to drive the point better.

What do you think?

**Comment by Bobeta** [comment]
@andresgalante Sure mate, I baked in the examples from the next paragraph so that we don't repeat ourselves

File: docs/4.0/getting-started/options.md [hunk 1]
```diff
@@ -8,13 +8,15 @@ toc: true
...
-Every Sass variable in Bootstrap 4 includes the `!default` flag, meaning you can override that default value in your own Sass even after that original variable's been defined. Copy and paste variables as needed, modify the values, remove the `!default` flag, and recompile.
-
-For example, to change out the `background-color` and `color` for the `<body>`, you'd do the following:
+Every Sass variable in Bootstrap 4 includes the `!default` flag, meaning you can override that default value in your own Sass. Copy and paste variables as needed, modify the values and remove the !default flag. If a variable has already been assigned, then it won't be re-assigned by the default values in Bootstrap. This means that your modified Sass variables should be called before you import Bootstrap Sass files. For example, to change out the `background-color` and `color` for the `<body>`, you'd do the following::
 
 {% highlight scss %}
-$body-bg:    $gray-900;
-$body-color: $gray-600;
+// Your variable overwrite first or a Sass file containing the modifications
+$body-bg: #000000;
```
Comment (andresgalante) [inline docs/4.0/getting-started/options.md]: I'd change this to `#000` and `#111` just to follow bootstrap [coding guidelines](http://codeguide.co/)

Comment (Bobeta) [reply docs/4.0/getting-started/options.md]: @andresgalante You're right, did that with the new commit

**Review by XhmikosR — APPROVED** [review]

**Comment by XhmikosR** [comment]
@Johann-S: LGTM. Please merge if you agree too.

**Pull request merged**